exercises/armstrong-numbers/armstrong_numbers_test.rb

```diff
@@ -0,0 +1,50 @@
+require 'minitest/autorun'
+require_relative 'armstrong_numbers'
+
+class ArmstrongNumbersTest < Minitest::Test
+  def test_one_digit_numbers_are_armstrong_numbers
+    assert ArmstrongNumbers.is_valid?(1)
```

Comment:
You'll need to add `skip`s to all the test methods (apart from the first) to be consistent with all the other tests.

Reply:
Done!